Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/VisualThemes/ThemeEngines/InteractableRotationTheme.cs

```diff
@@ -36,7 +37,15 @@ public override ThemeDefinition GetDefaultThemeDefinition()
                         Default = new ThemePropertyValue() { Vector3 = Vector3.zero }
                     },
                 },
-                CustomProperties = new List<ThemeProperty>(),
+                CustomProperties = new List<ThemeProperty>()
+                {
+                    new ThemeProperty()
+                    {
+                        Name = "Relative Rotation",
```

Comment:
The relative scale is a nice way to add backward compatibility, thank you for this. It would be good to document the meaning of "relative rotation" either in the interactable docs or if possible in a tooltip.

Reply:
i added the possibility to add a tooltip to custom properties.

Reply:
Thank you!